MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh

```diff
@@ -3,9 +3,10 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-export no_proxy=${your_no_proxy}
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
+export no_proxy=${no_proxy}
```

Comment:
Did you mean to change these? I think they have the `your_` prefix in the whole rest of the repo.

Reply:
I did for my own use, did not necessarily mean to contribute although I think it makes way more sense this way